package.json

```diff
@@ -29,22 +30,15 @@
     "chromatic": "npx chromatic --project-token dqq8k95raoo"
   },
   "peerDependencies": {
-    "emotion-theming": "^10.0.27",
     "react": "^16.13.1",
-    "react-datepicker": "^3.3.0",
-    "react-dom": "^16.13.1",
-    "rebass": "^4.0.7"
```

Comment:
our whole quartz is built on top of rebass. Added it to simple deps

Reply:
same as emotion-theming it's better to keep it as peer dependency to no having package duplication

Reply:
1. `react-datepicker`, we have a `DatePicker` component. And Quartz is a UI library. We shouldn't make our users install `react-datepicker` to use our `DatePicker`. It's totally wrong. Opposite: `hopsworks` should not use `react-datepicker`
2. `rebass` is a UI library. `quartz` is a UI library built on top of `quartz`. We shouldn't depend on `rebass` in `hopsworks`, thus we shouldn't install it there, thus `rebass` shouldn't be a peer dependency here

Reply:
I meant `rebass`, if we are not using rebass directly in hopsworks that's fine otherwise I think having it in peerdependacies will just install one instance of the library which preferred way to keep the bundle size minimal

Reply:
we're not using `rebass` directly in `hopsworks`

and we're heavily building on top of it
so I am 100% sure it should be a dep here